Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -613,18 +613,9 @@ AActor* ABenchmarkGymGameMode::FindPlayerStart_Implementation(AController* Playe
 		return SpawnManager->GetSpawnPointActorByIndex(PlayersSpawned);
 	}
 
-	// Use custom spawning with density controls
-	const int32 PlayerUniqueID = Player->GetUniqueID();
-	AActor** SpawnPoint = PlayerIdToSpawnPointMap.Find(PlayerUniqueID);
-	if (SpawnPoint != nullptr)
-	{
-		return *SpawnPoint;
```

Comment:
iirc FindPlayerStart_Implementation is called by SpatialSpawner to determine worker zone and then again during actual spawning logic, is this still a problem?

Reply:
Yeah it is, but sets Player to nullptr to indicate it isn't a "real" player spawning instance. And we special case for that just above this code here.